manifests/server/conf.pp

```diff
@@ -198,67 +220,84 @@
-  Boolean                          $firewall                        = simplib::lookup('simp_options::firewall', { 'default_value' => false }),
-  Boolean                          $haveged                         = simplib::lookup('simp_options::haveged', { 'default_value' => false }),
-  Boolean                          $ldap                            = simplib::lookup('simp_options::ldap', { 'default_value' => false }),
-  Boolean                          $oath                            = simplib::lookup('simp_options::oath', { 'default_value' => false }),
-  Boolean                          $manage_pam_sshd                 = $oath,
-  Integer[0]                       $oath_window                     = 1,
-  Variant[Enum['simp'],Boolean]    $pki                             = simplib::lookup('simp_options::pki', { 'default_value' => false }),
-  Boolean                          $sssd                            = simplib::lookup('simp_options::sssd', { 'default_value' => false }),
-  Simplib::Netlist                 $trusted_nets                    = ['ALL']
+  String                             $app_pki_external_source         = simplib::lookup('simp_options::pki::source', { 'default_value' => '/etc/pki/simp/x509' }),
+  Stdlib::Absolutepath               $app_pki_key                     = "/etc/pki/simp_apps/sshd/x509/private/${facts['fqdn']}.pem",
+  Boolean                            $enable_fallback_ciphers         = true,
+  Array[String]                      $fallback_ciphers                = $ssh::server::params::fallback_ciphers,
+  Boolean                            $fips                            = simplib::lookup('simp_options::fips', { 'default_value' => false }),
+  Boolean                            $firewall                        = simplib::lookup('simp_options::firewall', { 'default_value' => false }),
+  Boolean                            $haveged                         = simplib::lookup('simp_options::haveged', { 'default_value' => false }),
+  Boolean                            $ldap                            = simplib::lookup('simp_options::ldap', { 'default_value' => false }),
+  Boolean                            $oath                            = simplib::lookup('simp_options::oath', { 'default_value' => false }),
+  Boolean                            $manage_pam_sshd                 = $oath,
+  Integer[0]                         $oath_window                     = 1,
+  Variant[Enum['simp'],Boolean]      $pki                             = simplib::lookup('simp_options::pki', { 'default_value' => false }),
+  Boolean                            $sssd                            = simplib::lookup('simp_options::sssd', { 'default_value' => false }),
+  Simplib::Netlist                   $trusted_nets                    = ['ALL'],
+  Optional[Hash[String[1],NotUndef]] $custom_entries                  = undef
```

Comment:
Could we keep the parameters alphabetized, please?

Reply:
In this case, I dropped it at the bottom because it's generally supposed to be ignored and would jump out as weird in a large list of actual sshd parameters. Moving it into the SSH section makes sense though.

Reply:
Maybe put it in the middle, with its own `#### Custom Parameters ####` header.